### PR TITLE
[IMP] Pos: iot six module part added to the six doc 

### PR DESCRIPTION
**Pull request by Ysoroko**
This PR adds the documentation for a new Pos IoT Six module which will soon replace the old Six module.

Related PRs:
16.0 Enterprise: https://github.com/odoo/enterprise/pull/47086
16.0 Odoo: https://github.com/odoo/odoo/pull/13566
Worldline-lib: https://github.com/odoo/worldline-lib/pull/4

task-3555605 (sub-task of task-3031744)

### CODE REVIEW

**Comment by jcs-odoo** [comment]
Requesting a review from @odoo/be-doc-review since the team @odoo/pos-doc-review isn't available at the moment.

**Comment by LoredanaLrpz** [comment]
Hey @Ysoroko , I've pushed the changes, thanks a lot for your work!

**Review by dade-odoo — CHANGES_REQUESTED** [review]
Hi @LoredanaLrpz and @Ysoroko ! Nice work on this doc. Overall it's really clear and will be super helpful. I made a few specific comments/edits for you. Feel free to discuss any of them if you have any questions :)

One point about the screenshots: I believe @jcs-odoo (feel free to correct me if I'm wrong here) had said that if we're updating/creating a doc from 16.0, we can go ahead and use 16.3 screenshots with the updated design so that the screenshots will have more longevity (provided that the layout isn't significantly different between 16.0 and 16.3).

File: content/applications/sales/point_of_sale/payment_methods/terminals/six.rst [hunk 1]
```diff
@@ -2,56 +2,83 @@
...
 
-.. image:: six/pos-reverse-payment.png
-   :align: center
-   :alt: The Reverse button on the PoS interface.
+.. note::
+   - Once your payment is processed, the type of card used and the transaction ID appear on the
+     payment record.
+   - The language used for error messages is the same as the Six terminal. Configure the terminal to
+     change the language or contact Six. by configuring the terminal (Contact Six)
+   - By default, the port used is `7784`.
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/payment_methods/terminals/six.rst:79]: "the port used by the Six terminal"

File: content/applications/sales/point_of_sale/payment_methods/terminals/six.rst [hunk 2]
```diff
@@ -2,56 +2,83 @@
...
+
+Configure the Payment Method
+----------------------------
+
+From your database, go to :menuselection:`Point of Sale --> Configuration --> Payment Methods` to
+create a new payment method specifically for Six. Click :guilabel:`New`, set the :guilabel:`Journal`
+field as :guilabel:`Bank`, the :guilabel:`Use a Payment Terminal` field as :guilabel:`SIX IOT`, and
+select the corresponding device in the :guilabel:`Payment Terminal Device` field.
 
 .. image:: six/new-payment-method.png
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/payment_methods/terminals/six.rst:61]: note from the test day: the boolean field is for more recent versions

**Review by dade-odoo — APPROVED** [review]
Looks good :) @LoredanaLrpz

**Comment by xpl-odoo** [comment]
@Ysoroko feel free to r+ whenever!
@robodoo delegate+

**Comment by Ysoroko** [comment]
@robodoo r+

**Pull request closed without merging**